lib/components/narrative/metro/metro-itinerary.tsx

```diff
@@ -309,7 +305,6 @@ class MetroItinerary extends NarrativeItinerary {
     )
 
     const firstTransitStop = getFirstTransitLegStop(itinerary)
-    const routeLegs = itinerary.legs.filter(removeInsignifigantWalkLegs)
```

Comment:
I'm nervous about removing this for all route blocks -- this is used to remove a walking leg icon from a say 3 minute walking, 20 minute transit, 2 minute walking itinerary

Reply:
This was the only place in the file it was used, which is why I took it out, but I think `removeInsignifigantWalkLegs` is used other places for what you're talking about!

Reply:
Yes unfortunately this change affects Walk + Transit itineraries as well. 3 minute walk legs are being shown when they shouldn't be

Reply:
Can we do this instead, to address Miles' comment (which I agree with)
```
const routeLegs = itinerary.legs.length > 1 ? itinerary.legs.filter(removeInsignifigantWalkLegs) : itinerary.legs
```

Reply:
Just want to clarify that the routeLegs in this file does not affect transit itineraries - Miles & I discussed this offline and I think that's making this thread confusing (for me at least!!) so apologies for bad transparency!